src/protobuf_impl.rs

```diff
@@ -79,8 +80,10 @@ impl Builder {
         }
 
         let desc_bytes = std::fs::read(&desc_file).unwrap();
-        let desc: protobuf::descriptor::FileDescriptorSet =
-            protobuf::parse_from_bytes(&desc_bytes).unwrap();
+        let mut desc = protobuf::descriptor::FileDescriptorSet::new();
+        desc.merge_from_bytes(&desc_bytes).unwrap();
+        desc.check_initialized().unwrap();
```

Comment:
Why need to check it?

Reply:
I just copy the implementation of `parse_from_bytes` here...

Reply:
How about `FileDescriptorSet::parse_from_bytes(&desc_bytes).unwrap()`?

Reply:
It doesn't compile on old version protobuf (the version TiKV currently uses). I think we can avoid making an incompatible change.